README.md

```diff
@@ -1,54 +1,117 @@
-# Grasshopper Parser
+# Grasshopper Address Parser
 
 US Address parser for the [Grasshopper](https://github.com/cfpb/grasshopper) project.
 Provides a simple microservice that breaks an address string into its components.
 
-## Installation
+## Requirements
 
-This service depends on the [Flask Microframework](http://flask.pocoo.org/) and the [US Address parsing library](https://github.com/datamade/usaddress)
-To install these dependencies, run the following from your Python environment:
+* [Flask](http://flask.pocoo.org/) - Python HTTP microframework
+* [US Address parsing library](https://github.com/datamade/usaddress) - Python
+    library for parsing US addresses using natural language processing (NLP) methods.
 
-```
-pip install usaddress
-pip install flask
-```
+## Build and Deploy
+
+grasshopper-parser is a standard Python app with few dependencies, making it
+quite simple to build and launch.
+
+### Locally
+
+1. Build it.
+
+        pip install -r requirements.txt
+
+1. Run it.
 
-## Running
+        python app/app.py
 
-The service can be run directly from the root directory by issuing the following command:
 
-`python app.py`
+### Docker
 
+The service can also be run on Docker.
 
-Alternatively, this microservice can be run with Docker:
+1. Build it.
 
-To build, run `docker build --rm --tag=hmda/grasshopper-parser .`
+        docker build --rm --tag=hmda/grasshopper-parser .
 
-A container can be started by running `docker run -ti -p 5000:5000 hmda/grasshopper-parser`
+1. Run it.
+
+        docker run -ti -p 5000:5000 hmda/grasshopper-parser
 
 ## Usage
-Send a POST request to /addresses/parse with a JSON object containing the addres to be parsed:
+
+The following resources are available.  All examples assume running on `localhost`, port `5000`.
+
+### `/status`
+
+Displays the current state of the API.
+
+#### Request
+
+    GET http://localhost:5000/status
+
+#### Response
 
 ```json
 {
-  "address": "1311 30th st washington dc 20007"
+    "host": "yourhost.local",
+    "status": "OK",
+    "time": "2015-05-06T19:14:19.304850+00:00",
+    "upSince": "2015-05-06T19:08:26.568966+00:00"
 }
+```
+
+### `/parse`
+
+The `/parse` resource is the heart of this API.  It parses a free-text address
+strings into their component parts. 
+
+The following options are available via both `HTTP GET` query paramaters or
+JSON attributes via `HTTP POST`.
+
+* **`address`:** Free-text address string to be parsed.
+
+* **`method`:** (Optional) [usaddress parsing method](http://usaddress.readthedocs.org/en/latest/#usage)
+    to be used to split `address` into its component parts.
+
+    Values:
+    * `parse` (Default)
+    * `tag`
```

Comment:
I'm not sure the `method` param will stick around long-term.  I'm guessing we'll figure out which one works the best for us, and stick with that.  I think `/parse` if the better resource name whichever method we go with long term, so I think I'd rather stick with it and keep the params to pick which method we want.

For some examples of addresses that parse differently with `parse` vs. `tag`, they give the following in the [usage](http://usaddress.readthedocs.org/en/latest/#usage) docs.

* `Robie House, 5757 South Woodlawn Avenue, Chicago, IL 60637`
* `State & Lake, Chicago` - don't forget to escape the amperstand with `%26` on this one.
* `P.O. Box 123, Chicago, IL`

You'll notice things like place names get merged into a single `BuildingName` attribute and commas are stripped when using `tag`.